examples/networkpolicy/README.md

```diff
@@ -2,4 +2,25 @@
 
 ## Overview
 
-1. **kubernetes-calico.json** - deploying and using [Kubernetes](../../docs/kubernetes.md) with Calico network policy.
+The kubernetes-calico deployment template enables Calico networking and policies for the ACS-engine cluster via `"networkPolicy": "calico"` being present inside the `kubernetesConfig`.
+
+```
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "networkPolicy": "calico"
+      }
+```
+
+This deploys the [v2.6.0 release](https://docs.projectcalico.org/v2.6/releases/) of [Kubernetes Datastore Install](https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/kubernetes-datastore/) version of calico with the "Calico policy-only with user-supplied networking" which supports kubernetes ingress policies and has some limitations as denoted on the referenced page.
```

Comment:
nit: `v2.6.1`

Reply:
Also, I don't think this sentence is 100% accurate. The v2.6 release of Calico is applied if using Kubernetes v1.7+. But  [a compatible calico manifest](https://github.com/Azure/acs-engine/blob/master/parts/kubernetesmasteraddons-calico-daemonset1.5.yaml) is applied automatically when using <v1.7.